scripts/setup-adapters.sh

```diff
@@ -54,33 +54,51 @@ function install_aws_deps {
 }
 
 function install_gcs-sdk-cpp {
-  # Install gcs dependencies
+  # Install gcs dependencies that cannot be installed via cmake's FetchContent
   # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
 
-  # abseil-cpp
-  github_checkout abseil/abseil-cpp 20240116.1 --depth 1
-  sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
-  cmake_install -DBUILD_SHARED_LIBS=OFF \
-    -DABSL_BUILD_TESTING=OFF
+  # abseil
+  github_checkout abseil/abseil-cpp 20240116.2 --depth 1
```

Comment:
I have always used because it clones faster

--depth <depth>
           Create a shallow clone with a history truncated to the specified number of commits.